ethdb/pebble/pebble.go

```diff
@@ -253,6 +285,10 @@ func New(file string, cache int, handles int, namespace string, readonly bool) (
 		// By setting the WALBytesPerSync, the cached WAL writes will be periodically
 		// flushed at the background if the accumulated size exceeds this threshold.
 		WALBytesPerSync: 5 * ethdb.IdealBatchSize,
+
+		// Reduce L0CompactionThreshold from the default 4 to 2 to trigger compactions more frequently,
+		// which helps reduce compaction debt from ~10GB to below 500MB.
```

Comment:
we can keep the more detail comment from https://github.com/ethereum/go-ethereum/pull/31988/files

```
		// L0CompactionThreshold specifies the number of L0 read-amplification
		// necessary to trigger an L0 compaction. It essentially refers to the
		// number of sub-levels at the L0. For each sub-level, it contains several
		// L0 files which are non-overlapping with each other, typically produced
		// by a single memory-table flush.
		//
		// The default value in Pebble is 4, which is a bit too large to have
		// the compaction debt as around 10GB. By reducing it to 2, the compaction
		// debt will be less than 1GB, but with more frequent compactions scheduled.
```